docs/integrations/sources/instagram-migrations.md

```diff
@@ -1,5 +1,70 @@
 # Instagram Migration Guide
 
+## Upgrading to 3.0.0
+
+Connector will be upgraded to API v18. Affected Streams and their corresponding changes are listed below:
```

Comment:
The Instagram connector has been upgrade to API v18 (following the deprecation of v11).  Connector will be upgraded to API v18. Affected Streams and their corresponding changes are listed below: